### PR TITLE
Added properties that can be used to adjust bloom/blur effects.

### PR DESCRIPTION
**Pull request by osinoleg**
Added properties that can be used to adjust bloom/blur effects.

### CODE REVIEW

File: cocos2d/CCEffectGaussianBlur.h [hunk 1]
```diff
@@ -11,8 +11,13 @@
 #if CC_ENABLE_EXPERIMENTAL_EFFECTS
 @interface CCEffectGaussianBlur : CCEffect
 
-@property (nonatomic) GLKVector2 blurDirection;
+// blurRadius number of pixels blur will extend to (6 is the maximum, because we are limited by the number
```
Comment (slembcke) [inline cocos2d/CCEffectGaussianBlur.h:14]: Copy paste error from the comment below I assume?

Comment (osinoleg) [inline cocos2d/CCEffectGaussianBlur.h:14]: yep :P

File: cocos2d/CCEffectBloom.m [hunk 2]
```diff
@@ -189,7 +203,7 @@ -(void)buildFragmentFunctions
     
     // Choose one?
     NSString* addativeBlending =  @"src + dst";
-    NSString* screenBlending = [NSString stringWithFormat:@"(src + dst) - ((src * dst) * %f)", _intensity];
+    NSString* screenBlending = @"(src + dst) - ((src * dst) * u_intensity)";
```
Comment (slembcke) [inline cocos2d/CCEffectBloom.m:206]: If you added a min(src, dst) blending mode you could have a "gloom" filter too.

Comment (osinoleg) [inline cocos2d/CCEffectBloom.m:206]: ooo, good idea!! :) I like having a whole new effect, with minimal effort. I'll throw that in with the next pull request.

**Pull request merged**